### ISSUE TITLE
ImageReader problem

### ISSUE THREAD

**Issue by lianshushu** [DrSleep/tensorflow-deeplab-resnet#82]
Hi，I doubt that the function read_images_from_disk() only read one image when I run the train.py cased the problem. Is that right?  thank you for your help

Caused by op u'create_inputs/batch', defined at:
  File "train.py", line 255, in <module>
    main()
  File "train.py", line 140, in main
    image_batch, label_batch = reader.dequeue(args.batch_size)
  File "/home/lianliu/Documents/program/tensorflow-deeplab-resnet/deeplab_resnet/image_reader.py", line 199, in dequeue
    num_elements)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/training/input.py", line 917, in batch
    name=name)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/training/input.py", line 712, in _batch
    dequeued = queue.dequeue_many(batch_size, name=name)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/ops/data_flow_ops.py", line 458, in dequeue_many
    self._queue_ref, n=n, component_types=self._dtypes, name=name)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/ops/gen_data_flow_ops.py", line 1328, in _queue_dequeue_many_v2
    timeout_ms=timeout_ms, name=name)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/framework/op_def_library.py", line 768, in apply_op
    op_def=op_def)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/framework/ops.py", line 2336, in create_op
    original_op=self._default_original_op, op_def=op_def)
  File "/usr/local/lib/python2.7/dist-packages/tensorflow/python/framework/ops.py", line 1228, in __init__
    self._traceback = _extract_stack()

OutOfRangeError (see above for traceback): FIFOQueue '_1_create_inputs/batch/fifo_queue' is closed and has insufficient elements (requested 10, current size 0)
	 [[Node: create_inputs/batch = QueueDequeueManyV2[component_types=[DT_FLOAT, DT_UINT8], timeout_ms=-1, _device="/job:localhost/replica:0/task:0/cpu:0"](create_inputs/batch/fifo_queue, create_inputs/batch/n)]]

**Comment by Yaredoh** [comment]
If you are working on your own dataset, the problem may be is related with image list reader or the label/ground truth that you are using. Follow this steps to resolve, if the problem is related with that. [https://gist.github.com/DrSleep/4bce37254c5900545e6b65f6a0858b9c](url)

**Comment by DrSleep** [comment]
@lianshushu , please check that your inputs do exist and are in appropriate format.
